backup.config-example

```diff
@@ -23,11 +23,13 @@ GHE_NUM_SNAPSHOTS=10
 #
 #GHE_RESTORE_HOST="github-standby.example.com"
 
-# Any extra options passed to the SSH command.  Nothing required by default
+# Any extra options passed to the SSH command.
+# In a single instance environment, nothing is required by default.
+# In a clustering environment, "-i path-to-ssh-private-key" is required.
```

Comment:
I would just replace with "full-path-to-ssh-private-key" as somehow, relative paths are not working